lms/static/scripts/frontend_apps/utils/test/onedrive-picker-client-test.js

```diff
@@ -0,0 +1,145 @@
+import { PickerCanceledError } from '../google-picker-client';
```

Comment:
If we're going to reuse this exception for both the Google and OneDrive picker we probably want to move it into a shared location in future.

Reply:
Agree.